apps/web/src/services/administration/iam/user/modules/user-management-modal/modules/UserInfoForm.vue

```diff
@@ -0,0 +1,280 @@
+} from '@/services/administration/iam/user/lib/user-form-validations';
+import type { User } from '@/services/administration/iam/user/type';
+import { useUserPageStore } from '@/services/administration/store/user-page-store';
+
+interface Props {
+    activeTab?: string;
+    item?: User;
+}
+const props = withDefaults(defineProps<Props>(), {
+    activeTab: '',
+    item: undefined,
+});
+
+const userPageStore = useUserPageStore();
+const userPageState = userPageStore.$state;
+
+const emit = defineEmits<{(e: 'change-input', formState): void}>();
+
+const state = reactive({
+    loading: false,
+    searchText: '',
+    selectedItems: [] as MenuItem[],
+    externalItems: [] as MenuItem[],
+    supportFind: computed(() => !!store.state.domain.authOptions?.support_find),
+    users: computed<UserReferenceMap>(() => store.getters['reference/userItems']),
+});
+const formState = reactive({
+    userId: '',
+    name: '',
+});
+const validationState = reactive({
+    isUserIdValid: undefined as undefined | boolean,
+    userIdInvalidText: '' as TranslateResult | string,
+    userIdValidText: '' as TranslateResult | string,
+});
+
+/* Components */
+const setExternalMenuItems = (users) => {
+    state.externalItems = [];
+    users.forEach((user) => {
+        const singleItem = {
+            name: user.userId,
+            label: user.name ? `${user.userId} (${user.name})` : user.userId,
+            disabled: false,
+        };
+        if (state.users[user.userId]) {
+            singleItem.label = `(${i18n.t('IDENTITY.USER.FORM.ALREADY_EXISTS')}) ${singleItem.label}`;
+            singleItem.disabled = true;
+        }
+        state.externalItems.push(singleItem);
+    });
+};
+const onDeleteSelectedExternalUser = () => {
+    formState.userId = '';
+    formState.name = '';
+    validationState.isUserIdValid = undefined;
+    validationState.userIdInvalidText = '';
+};
+const onSelectExternalUser = async (userItem) => {
+    await getExternalUser(userItem.name);
+    await handleClickCheckId();
+};
+const handleChangeInput = () => {
+    emit('change-input', { ...formState, name: formState.name });
+};
+const setForm = () => {
+    formState.userId = props.item.user_id;
+    formState.name = props.item.name || '';
+};
+
+/* API */
+const listExternalUser = debounce(async () => {
+    try {
+        state.loading = true;
+        const { results } = await SpaceConnector.client.identity.user.find({
+            search: {
+                keyword: state.searchText,
+            },
+        });
+        await setExternalMenuItems(results);
+    } catch (e) {
+        ErrorHandler.handleError(e);
+        state.externalItems = [];
+    } finally {
+        state.loading = false;
+    }
+}, 300);
+const getExternalUser = async (userId: string) => {
+    try {
+        const { results } = await SpaceConnector.client.identity.user.find({
+            search: {
+                userId,
+            },
+        });
+        if (results.length) {
```

Comment:
suggestion: I think this will be more intuitive!

```suggestion
        const { results, total_count } = await SpaceConnector.client.identity.user.find({
            search: {
                userId,
            },
        });
        if (total_count > 0) {
```